src/utilities/prTitleParsers.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I don't see why we'd disable it instead of improving the code?

Reply:
The reason I ended up using non-null assertions was because of this: https://github.com/microsoft/TypeScript/issues/32098

So in my case I have:

```typescript
const regExp = /bump .* from (?<from>.*) to (?<to>.*)/u;
const match = regExp.exec(title);

// match can be null
if (match === null) {
  return true;
}

// but if match is not null there is no way that the `from` and `to` don't exist
// tho Typescript is not smart enough to understand that
```

But if you rather have me not have the `@typescript-eslint/no-non-null-assertion` then I can check the `undefined`ness of the `from` and `to` tho the coe would become uglier. WDYT?

Reply:
That seems like a valid reason, thanks for the explanation!